src/components/editions/teamScore/index.tsx

```diff
@@ -86,27 +87,30 @@ const scorerStyles = (location: TeamLocation): SerializedStyles => css`
 	}
 `;
 
-const TeamScore: FC<Props> = ({ team, location }) => (
-	<section css={styles(location)}>
-		<div css={scoreStyles(location)}>
-			<div css={scoreNumberStyles}>
-				<span css={scoreInlineStyles}>{team.score}</span>
+const TeamScore: FC<Props> = ({ team, location }) => {
+	const scorers = fromNullable(team.scorers);
+	return (
+		<section css={styles(location)}>
+			<div css={scoreStyles(location)}>
+				<div css={scoreNumberStyles}>
+					<span css={scoreInlineStyles}>{team.score}</span>
+				</div>
 			</div>
-		</div>
-		<div css={infoStyles(location)}>
-			<h3 css={teamNameStyles}>{team.name}</h3>
-			{team.scorers.length > 0 && (
-				<ul css={scorerStyles(location)}>
-					{team.scorers.map((scorer) => (
-						<li key={`${scorer.player}`}>
-							{scorer.player} {scorer.timeInMinutes}&apos;{' '}
-							{scorer.additionalInfo}
-						</li>
-					))}
-				</ul>
-			)}
-		</div>
-	</section>
-);
+			<div css={infoStyles(location)}>
+				<h3 css={teamNameStyles}>{team.name}</h3>
+				{scorers.kind === OptionKind.Some && (
```

Comment:
You can use `maybeRender` for this if it's now an `Option`:

```tsx
{maybeRender(scorers, s =>
  <ul ...>
    {s.map(scorer => ...)}
  </ul>
)}
```

Reply:
Good call.